### DISCUSSION TITLE
Patient vs population

### DISCUSSION THREAD

**Discussion by daniellebitt** [TheLightCollective/PatientLedAI#8] [Ideas]
It might be helpful to define the patient population - in essence everyone is, has been, or will be a patient and so these rights are therefore human rights. But as you rightfully highlight when in the patient position people are particularly vulnerable to harms.

**Comment by AndreaDowning** [comment]
Thank you this is great feedback @daniellebitt !  As we close comments out in 8 days, we'll ask the team to review this in the language / update.  

One key distinction here when we talk about patient representation is that while everyone is, has been, or will be a patient ....there is a big and important difference for those who a) live with disabilities/chonic conditions or b) who work to organize communities, advocate for research, or build a professional set of skills around this work to advance patient rights.  

As you point out it's super important for us to define _both_ in this document and note the distinction, especially when we talk about having an independent fiduciary role to patients.   It's also worth noting this is a new emerging area of study.  For example:

1 -  We recently got to collaborate on this great project about individual vs. community representation with respect to patients:  https://sagebionetworks.pubpub.org/pub/pruyy6rf/release/1

2 - I love this resource from PCORI which [established measures](https://www.pcori.org/resources/measuring-what-matters-advancing-science-and-practice-engagement) for patient/community engagement.
